vue-frontend/src/components/jobs/CareerView.vue

```diff
@@ -0,0 +1,291 @@
+    },
+    getCareer() {
+      axios
+        .get(config.API_BASE + "/api/careers")
+        .then((response) => {
+          this.careers = response.data;
+          this.careers[0].icon_name = faApple;
+          this.careers[1].icon_name = faAndroid;
+          this.careers[2].icon_name = faGlobe;
+          this.careers[3].icon_name = faBullhorn;
+          this.careers[4].icon_name = faPenNib;
+          this.careers[5].icon_name = faApple;
+        })
+        .catch((error) => {
+          console.log(error);
+        });
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.container {
+  margin: 100px auto;
+}
+
+.description {
+  margin: 30px 10px;
+  color: #3d3d3d;
+}
+
+.text-weight {
+  font-weight: 700;
+}
+
+.career-container {
+  border-radius: 0.9rem;
+  border: 1px solid #e2e2e2;
+  overflow: hidden;
+  margin: 30px 10px;
+}
+
+.career-list-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.career-list {
+  display: flex;
+  flex-direction: column;
+}
+
+.career-divider {
+  background: #e2e2e2;
+  height: 1px;
+}
+
+.career-header {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  align-items: center;
+  padding: 30px;
+  cursor: pointer;
+}
+
+.career-icon {
+  width: 2.3rem;
+}
+
+.career-summary {
+  padding: 30px;
+}
+
+.career-title {
+  font-style: normal;
+  font-weight: 700;
+  color: #3d3d3d;
+}
+
+.career-plus-icon {
+  margin-left: auto;
+  order: 2;
+  color: rgba(61, 61, 61, 0.5);
+}
+
+.read-apply-btns {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 0 0 30px;
+}
+
+.gradient-btn {
+  margin: 5px;
+}
+
+.gradient-btn > span {
+  font-size: 1.125rem;
+  line-height: 1.364rem;
+}
+
+.gradient-career-icon {
+  color: #f2709c;
+}
+
+.gradient-border-btn {
+  border-radius: 0.6rem;
+  padding: 1rem;
+  margin: 5px;
+  text-align: center;
+  border: 2px solid transparent !important;
+  background: linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 0)),
+    linear-gradient(to top, #ff9472, #f2709c);
+  box-shadow: 2px 1000px 1px #fff inset;
+}
+
+.gradient-border-btn > span {
+  margin: 0 6px;
+  font-weight: 700;
+  color: rgba(61, 61, 61, 1);
+  font-size: 1.125rem;
+  line-height: 1.364rem;
+}
+
+@media (hover: hover) and (pointer: fine) {
+  .gradient-border-btn:hover {
+    background: linear-gradient(91.53deg, #f2709c 3.91%, #ff9472 100%);
+    border: 2px solid transparent;
+    box-shadow: none;
+  }
+
+  .gradient-border-btn:hover > span {
+    -webkit-text-fill-color: white;
+  }
+
+  .gradient-border-btn:hover > .fa {
+    color: white;
+  }
+}
+
+@include media-breakpoint-up(sm) {
+  .read-apply-btns {
+    justify-content: flex-end;
+    padding: 0 20px 30px;
+  }
+}
+
+@include media-breakpoint-up(md) {
+  .career-header {
+    padding: 32px 48px;
+  }
+
+  .career-summary {
+    padding: 32px 48px;
+  }
+
+  .description {
+    margin: 50px 80px;
+  }
+
+  .career-container {
+    margin: 0 80px;
```

Comment:
Only top-bottom margin should be fix, We can take left-right margins and padding in %